lib/fog/azurerm/docs/network.md

```diff
@@ -26,14 +26,19 @@ Next, create a connection to the Network Service:
 Create a new virtual network
 
 ```ruby
-    vnet = azure_network_service.virtual_networks.create(
-        name: '<Virtual Network name>',
-        location:         'westus',
-        resource_group: '<Resource Group name>',
-        subnet_address_list: '10.1.0.0/24',  
-        dns_list: '10.1.0.5,10.1.0.6',
-        network_address_list: '10.1.0.0/16,10.2.0.0/16'   
- )
+    vnet = network.virtual_networks.create(
+      name:             '<Virtual Network Name>',
+      location:         'westus',
+      resource_group:   '<Resource Group Name>',
+      subnets:          [{
+        name: '<Subnet Name>',
+        address_prefix: '10.1.0.0/24',
+        network_security_group_id: '/subscriptions/######/resourceGroups/fog-rg/providers/Microsoft.Network/networkSecurityGroups/fognsg',
```

Comment:
use place holders for <subscription-id>, <resource-group> and <network-security-group>